streaming/base/converters/dataframe_to_mds.py

```diff
@@ -287,6 +291,17 @@ def write_mds(iterator: Iterable):
 
     # Fix output format as mds_path: Tuple(local, remote)
     if cu.remote is None:
+        # If dataframe_to_mds is being called, this is in a distributed Spark env.
+        # If there is no remote, it's because the given out path is local, which does not 
+        # in general make sense, unless this 'local' path is FUSE-mounted distributed 
+        # storage such as /dbfs or /Volumes in Databricks for example.
+        # It's not wrong in this case, but probably nevertheless desirable to specify a local temp
+        # path explicitly, to interpret the FUSE-mounted path as remote
```

Comment:
```suggestion
        # path explicitly to interpret the FUSE-mounted path as remote
```

Reply:
I think the comma correctly adds clarity, and is not wrong in any event

Reply:
Cool with me, just a minor nit. Not gonna nitpick on grammar too much haha (i am trying to cut my own overuse of commas)